test/end-to-end/pages/thankyou/thankyou.js

```diff
@@ -1,12 +1,16 @@
 'use strict';
 
-const thankYouContent = require('app/resources/en/translation/thankyou');
+const thankYouContentEn = require('app/resources/en/translation/thankyou');
+const thankYouContentCy = require('app/resources/cy/translation/thankyou');
 
-module.exports = async function() {
+module.exports = async function(language ='en') {
     const I = this;
+    const thankYouContent = language === 'en' ? thankYouContentEn : thankYouContentCy;
 
     await I.checkPageUrl('app/steps/ui/thankyou');
-    await I.waitForText(thankYouContent.header);
+    if (language === 'en') {
+        await I.waitForText(thankYouContent.header);
+    }
```

Comment:
again, not sure why you'd only want to check the text here on English journeys? It will likely help with test stability for both languages